packages/circuits/scripts/buildSnarks.mjs

```diff
@@ -24,35 +26,46 @@ for (const name of circuits) {
     if (!circuitContents[name])
         throw new Error(`Unknown circuit name: "${name}"`)
 
-    await fs.promises.writeFile(
-        path.join(outDir, `${name}_main.circom`),
-        circuitContents[name]
-    )
-
-    const inputFile = path.join(outDir, `${name}_main.circom`)
+    const inputFileBuild = path.join(buildDir, `${name}_main.circom`)
```

Comment:
This variable shouldn't be needed. It's only used in a log below and can be replaced with `inputFileOut`

Reply:
Yep, using `inputFileOut` now